influxdb/_dataframe_client.py

```diff
@@ -131,25 +132,26 @@ def write_points(self,
                 protocol=protocol)
             return True
 
-    def query(self, query, chunked=False, database=None):
+    def query(self, query, dropna=True, **kwargs):
         """
         Quering data into a DataFrame.
 
-        :param chunked: [Optional, default=False] True if the data shall be
-            retrieved in chunks, False otherwise.
+        :param query: the actual query string
+        :param dropna: drop columns where all values are missing
+        :param **kwargs: additional parameters for ``InfluxDBClient.query``
 
         """
-        results = super(DataFrameClient, self).query(query, database=database)
+        results = super(DataFrameClient, self).query(query, **kwargs)
```

Comment:
The rationale behind using `kwargs` is just to make the code more compact and maintainable.

 `DataFrameClient.query` is nothing more than a wrapper around `InfluxDBClient.query`. I personally think wrappers should be as thin as possible and do not see the need to be more verbose and copy-and-paste all the extra optional arguments. Adding all the optional arguments of `InfluxDBClient.query` to `DataFrameClient.query` will also make future maintenance more difficult and error prone. Whenever there is some change in the signature of `InfluxDBClient.query`, that change will need to be manually reflected to all wrapper functions.

This `kwargs` approach is widely used in libraries that have a lot of functions wrapping other functions with a large number of arguments. Matplotlib is the best example that comes up to my mind.

As for passing inexistent arguments causing a runtime error, well, the same can be said for passing inexistent arguments directly to `InfluxDBClient.query`. The only REAL disadvantage of the `kwargs` argument I can think of is the lack of IDE tab completion help. 

So, IMHO, it boils down to a trade off between maintainability+compactness vs IDE completion. I personally think the former to be way more important, but I will commit some code reflecting your change request.